disnake/ui/action_row.py

```diff
@@ -269,7 +269,7 @@ def add_button(
     def add_select(
         self: Union[
             ActionRow[MessageUIComponent],
-            # ActionRow[ModalUIComponent],
+            ActionRow[ModalUIComponent],
```

Comment:
The docstring still shows `Can only be used if the action row holds message components`, we can just remove that section now, imo.

Reply:
https://github.com/DisnakeDev/disnake/pull/558/commits/31d2e119bdf7c5597c4d3af2f41dc0e3d4f6eee3